x-pack/test/detection_engine_api_integration/security_and_spaces/rule_execution_logic/query.ts

```diff
@@ -814,7 +815,7 @@ export default ({ getService }: FtrProviderContext) => {
             [ALERT_ORIGINAL_TIME]: firstTimestamp,
             [ALERT_SUPPRESSION_START]: firstTimestamp,
             [ALERT_SUPPRESSION_END]: secondTimestamp,
-            [ALERT_SUPPRESSION_DOCS_COUNT]: 3,
+            [ALERT_SUPPRESSION_DOCS_COUNT]: 5,
```

Comment:
Detection rules set `autoRecoverAlerts` to false when registering their rule type so they are not lifecycle alerts and their alerts should not be marked as untracked when the rule is disabled. I tested this in the PR and the alerts are not marked as untracked, so not sure why this would change in this PR

Reply:
@ymao1 Moving this logic into the task manager's `bulkDisable` method bypasses the `autoRecoverAlerts` check. I can try to restore it there, though it's difficult given that rule types are fetched within the OCC function and the tasks get disabled outside of it.

Reply:
Apparently this wasn't the issue? Test is still failing after I tried adding an `autoRecoverAlerts` check

Reply:
Reverted to `cbb17c1` and then merged with latest `main` and this test still fails. Definitely not related to this PR